composer.json

```diff
@@ -26,14 +26,17 @@
     "require-dev": {
         "friendsofsymfony/rest-bundle": "^1.1",
         "jms/serializer-bundle": "~0.11 || ^1.0",
+        "sonata-project/block-bundle": "~3.0",
         "sonata-project/media-bundle": "^3.0@dev",
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "suggest": {
+        "sonata-project/block-bundle": "For rendering dynamic list blocks on a page.",
         "sonata-project/media-bundle": "For media management"
     },
     "conflict": {
-        "jms/serializer": "<0.13"
+        "jms/serializer": "<0.13",
+        "sonata-project/block-bundle": "<3.0 || >=4.0"
```

Comment:
Your are talking on the `conflict` section. The logic is reversed.

https://getcomposer.org/doc/04-schema.md#conflict